modules/ROOT/pages/managing-kubernetes.adoc

```diff
@@ -1,5 +1,65 @@
 = Managing a Sync Gateway Cluster
 
+== Troubleshooting and Log Collection
+
+This section provides information about how to diagnose and troubleshoot problems with the Sync Gateway deployment.
+When troubleshooting, it is important to rule out Kubernetes itself as the root cause of the problem you are experiencing. See the Kubernetes [Troubleshooting](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/) Guide for information about debugging applications within a Kubernetes cluster.
+For troubleshooting Couchbase server deployment using Couchbase operator, please refer to our [guides](https://docs.couchbase.com/operator/1.1/logs-troubleshooting.html)
+
+=== Prerequisites
++ The location of the Sync Gateway logs within a pod is specified with the “logging” key in the Sync Gateway config file (https://docs.couchbase.com/sync-gateway/2.1/logging.html#log-rotation-configuration)
++ To view or fetch the logs from a Sync Gateway pod, you must first identify the Sync Gateway pods in your deployment using the following command. 
++
+`kubectl get pods`
+Your response will include a list of sync gateway pods in your deployment
+
+===  Viewing Logs in Console
+You can use kubectl to view logs for a specific pod using following command. This command will output logs to the console
+`kubectl logs -f <pod_id>`
+
+===  Using sgcollect_info
+[sgcollect_info] (https://docs.couchbase.com/sync-gateway/2.1/sgcollect-info.html) is command line utility that detailed statistics for a specific Sync Gateway node. This tool must be run on each node individually.
+[Download] (https://docs.couchbase.com/sync-gateway/2.1/getting-started.html)  the tool for the platform on which your pod was deployed. The tool is bundled with platform specific Sync Gateway binary 
```

Comment:
Don't think we need to talk about the download, since it's bundled inside the docker image

Reply:
Great point. I completely missed that!
```suggestion
+ The sgcollect-info tool is bundled as part of the Sync Gateway docker image 
```